pkg/plugins/manifest_test.go

```diff
@@ -3,10 +3,43 @@ package plugins
 import (
 	"strings"
 	"testing"
+	"time"
+
+	"github.com/grafana/grafana/pkg/services/sqlstore"
 
 	"github.com/stretchr/testify/assert"
 )
 
+func TestDatabaseAccessForManifestKeys(t *testing.T) {
+	testDB := sqlstore.InitTestDB(t)
+	mv := newManifestVerifier(testDB)
+
+	nowEpoch := time.Now().Unix()
+	testKeys := []ManifestKeys{
+		{KeyID: "keyid1", PublicKey: "publicKey1", UpdatedAt: nowEpoch, Since: nowEpoch},
+		{KeyID: "keyid2", PublicKey: "publicKey2", UpdatedAt: nowEpoch, Since: nowEpoch},
+	}
+
+	t.Run("can write keys to database", func(t *testing.T) {
+		session := mv.sqlstore.NewSession()
+		defer session.Close()
+		affected, err := session.Insert(testKeys)
+		assert.Nil(t, err)
+		assert.Equal(t, affected, int64(len(testKeys)))
```

Comment:
The expectation comes before what you're checking in Equal:
```suggestion
		assert.Equal(t, int64(len(testKeys)), affected)
```